packages/hdwallet-native/src/crypto/isolation/adapters/binance.ts

```diff
@@ -3,15 +3,15 @@ import { BIP32, Digest, SecP256K1 } from "../core"
 
 type SigningDelegate = Parameters<BncClient["setSigningDelegate"]>[0];
 
-export function signingDelegate(keyPair: BIP32.Node): SigningDelegate {
+export default {
+  async create(keyPair: BIP32.Node): Promise<SigningDelegate> {
```

Comment:
This helps normalize the way that isolation adapters are used -- via `await Isolation.Adapters.Binance.create()` just like any other isolation adapter.